src/peer_digest.cc

```diff
@@ -65,19 +66,10 @@ static const time_t GlobDigestReqMinGap = 1 * 60;   /* seconds */
 
 static time_t pd_last_req_time = 0; /* last call to Check */
 
-PeerDigest::PeerDigest(CachePeer * p)
+PeerDigest::PeerDigest(CachePeer *p) :
+    peer(p),
+    host(peer->host) // if peer disappears, we will know it's name
```

Comment:
May as well fix the grammar since we are moving/adjusting that comment:

```suggestion
    host(peer->host) // if peer disappears, we will know its name
```